monitoring/mock_uss/scdsc/routes_injection.py

```diff
@@ -70,8 +77,21 @@ def query_operational_intents(
 @requires_scope([SCOPE_SCD_QUALIFIER_INJECT])
 def scdsc_injection_status() -> Tuple[str, int]:
     """Implements USS status in SCD automated testing injection API."""
-    return flask.jsonify({"status": "Ready", "version": versioning.get_code_version()})
+    return flask.jsonify({'status': 'Ready', 'version': versioning.get_code_version()})
+
+@webapp.route('/scdsc/v1/startreport', methods=['POST'])
+@requires_scope([SCOPE_SCD_QUALIFIER_INJECT])
+def scdsc_injection_start_reporter() -> Tuple[str, int]:
```

Comment:
The `scdsc` functionality is meant to support the scd.yaml interface.  Since these endpoints are not part of that interface (and often not used when using that interface's set of capabilities), let's put these endpoints in a new/different set of mock_uss capabilities -- perhaps named `msgsigning` or similar.

Reply:
Will do.